### DISCUSSION TITLE
Spacing of components and content

### DISCUSSION THREAD

**Discussion by ukozdan** [zendeskgarden/react-components#1125] [Q&A]
Perhaps I'm missing it in the documentation, but how does Zendesk Garden deal with vertical spacing between components, i.e. margin? For instance, for a series of vertically stacked form inputs, how do I add vertical space between them without introducing custom CSS? Is there a need for a spacing module?

**Comment by jzempel** [comment]
Currently Garden's [Grid](https://garden.zendesk.com/components/grid) component does not provide vertical spacing props. This may be an option we explore in the future. For now vertical spacing is left as an exercise for the consumer.
